test/viewmodels/not_found_viewmodel_test.dart

```diff
@@ -0,0 +1,50 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+
+// SERVICES / MANAGERS
+import 'package:notredame/core/services/navigation_service.dart';
+
+// VIEW MODEL
+import 'package:notredame/core/viewmodels/not_found_viewmodel.dart';
+
+// OTHER
+import 'package:notredame/core/constants/router_paths.dart';
+import '../helpers.dart';
+
+void main() {
+  NavigationService navigationService;
+
+  NotFoundViewModel viewModel;
+
+  group('NotFoundViewModel - ', () {
+    const String _pageNotFoundPassed = "/test";
+    setUp(() async {
+      navigationService = setupNavigationServiceMock();
+      setupAnalyticsServiceMock();
+      setupLogger();
+
+      viewModel = NotFoundViewModel(_pageNotFoundPassed);
```

Comment:
A test should be added to validate if the viewmodel really call the analyticsService with the right parameters 😄